go/vt/vtgate/router_select_test.go

```diff
@@ -505,6 +506,35 @@ func TestSelectIN(t *testing.T) {
 
 	sbc1.Queries = nil
 	sbc2.Queries = nil
+	_, err = routerExec(router, "select id from user where id in ::vals", map[string]interface{}{
+		"vals": []interface{}{int64(1), int64(3)},
+	})
+	if err != nil {
+		t.Error(err)
+	}
+	wantQueries = []querytypes.BoundQuery{{
+		Sql: "select id from user where id in ::__vals",
+		BindVariables: map[string]interface{}{
+			"__vals": []interface{}{int64(1)},
+			"vals":   []interface{}{int64(1), int64(3)},
```

Comment:
I took "vals" to be an example of other bind variables that would normally be related to the query. In that case, maybe this code should actually pass in something that makes sense to the reader like a limit_count.